examples/simplehttp/common/httpclient.go

```diff
@@ -0,0 +1,69 @@
+package common
+
+import (
+	"context"
+	"errors"
+	"io"
+	"net/http"
+	"syscall"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	cancelTimeout = 2 * time.Second
+)
+
+// Post ...
+func Post(address string, e cloudevents.Event) (int, error) {
+	sendCtx, sendCancel := context.WithTimeout(context.Background(), cancelTimeout)
+	defer sendCancel()
+	p, err := cloudevents.NewHTTP()
+	if err != nil {
+		log.Errorf("failed to create protocol: %s", err.Error())
+		return http.StatusBadRequest, err
+	}
+	c, err := cloudevents.NewClient(p, cloudevents.WithUUIDs(), cloudevents.WithTimeNow())
```

Comment:
Is this c=NewClient object cleaned up somewhere?

Reply:
provided transport object and func scoped